nautobot/docs/installation/services.md

```diff
@@ -168,6 +168,34 @@ PrivateTmp=true
 WantedBy=multi-user.target
 ```
 
+Additionally, paste the following into `/etc/systemd/system/nautobot-beat-worker.service`:
+
+```
+[Unit]
+Description=Nautobot Celery Beat Worker
+Documentation=https://nautobot.readthedocs.io/en/stable/
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+Type=exec
+Environment="NAUTOBOT_ROOT=/opt/nautobot"
+
+User=nautobot
+Group=nautobot
+PIDFile=/var/tmp/nautobot-beat-worker.pid
+WorkingDirectory=/opt/nautobot
+
+ExecStart=/opt/nautobot/.local/bin/nautobot-server celery beat --loglevel INFO --pidfile /var/tmp/nautobot-beat-worker.pid --scheduler nautobot.core.celery.schedulers:NautobotDatabaseScheduler
```

Comment:
Since we're setting `CELERY_BEAT_SCHEDULER` in `settings.py` it shouldn't be necessary to set `--scheduler nautobot.core.celery.schedulers:NautobotDatabaseScheduler` here, and it probably *shouldn't* be set here in case we want or need to change the default later.